backend/src/packages/users/user.entity.ts

```diff
@@ -3,82 +3,119 @@ import { type Entity } from '#libs/types/types.js';
 class UserEntity implements Entity {
   private id: number | null;
 
+  private createdAt: string | null;
+
+  private updatedAt: string | null;
```

Comment:
createdAt/updatedAt should have Date type, as Yelyzaveta mentioned [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/issues/12#issuecomment-1687974980)

Reply:
Ok. thanks